src/include/miopen/solver.hpp

```diff
@@ -1492,6 +1492,76 @@ extern template struct ConvMPBidirectWinograd<4, 3>;
 extern template struct ConvMPBidirectWinograd<5, 3>;
 extern template struct ConvMPBidirectWinograd<6, 3>;
 
+template <int WinoDataH, int WinoFilterH, int WinoDataW = WinoDataH, int WinoFilterW = WinoFilterH>
+struct ConvMPBidirectWinograd_xdlops : SolverBase<ConvolutionContext>
+{
+    bool IsApplicable(const ConvolutionContext& ctx) const;
+
+    bool IsValidPerformanceConfig(const ConvolutionContext& ctx,
+                                  const PerformanceImplicitGemmForwardV4R4Xdlops& c) const
+    {
+        return ConvHipImplicitGemmForwardV4R4Xdlops{}.IsValidPerformanceConfig(
+            GetTransformedConvContext(ctx), c);
+    }
+
+    size_t GetWorkspaceSize(const ConvolutionContext& ctx) const
+    {
+        return ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>()
+                   .GetWorkspaceSize(ctx) +
+               ConvHipImplicitGemmForwardV4R4Xdlops{}.GetWorkspaceSize(
+                   GetTransformedConvContext(ctx));
+    }
+
+    ConvSolution GetSolution(const ConvolutionContext& ctx,
+                             const PerformanceImplicitGemmForwardV4R4Xdlops& config,
+                             bool disableConfigOverrideFromEnv = false) const;
+
+    ConvolutionContext GetTransformedConvContext(const ConvolutionContext&) const;
+
+    // kernel_file_name for solver identification
+    static std::string GetSolverFileNames(int id)
+    {
+        return ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::
+            GetSolverFileNames(id);
+    }
+
+    static std::string GetSolverKernelNames(int id)
+    {
+        return ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::
+            GetSolverKernelNames(id);
+    }
+
+    static int GetSolverWinoXformHWSize()
+    {
+        return ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::
+            GetSolverWinoXformHWSize();
+    }
+
+    PerformanceImplicitGemmForwardV4R4Xdlops
+    GetPerformanceConfig(const ConvolutionContext& ctx) const
+    {
+        return ConvHipImplicitGemmForwardV4R4Xdlops{}.GetPerformanceConfig(
+            GetTransformedConvContext(ctx));
+    }
+
+    PerformanceImplicitGemmForwardV4R4Xdlops Search(const ConvolutionContext&) const;
+
+    template <typename B, typename T>
+    int RunAndMeasureSolution(const miopen::Handle& handle,
+                              B bot_buf,
+                              T top_buf,
+                              ConstData_t wei_buf,
+                              ConstData_t bias_buf,
+                              const ConvolutionContext& ctx,
+                              const ConvSolution& solution,
+                              float& elapsed_time) const;
+};
+extern template struct ConvMPBidirectWinograd_xdlops<2, 3>;
```

Comment:
`<2,3>` is not needed.

Reply:
Do you know any other winograd2x3 wtih xdlops? Or are there other reasons?

Reply:
Needed! We don't have any other winograd f(2,3)  with xdlops

Reply:
But is wasn't instantiated. Now I see you've removed this declaration.